src/dashboard/Data/Browser/BrowserTable.react.js

```diff
@@ -270,6 +270,8 @@ export default class BrowserTable extends React.Component {
                 <EmptyState
                   title='No data to display'
                   description='This relation has no rows'
+                  cta='Add a row'
+                  action={this.props.onAddRow}
```

Comment:
This button correctly works adding a row to the relation. I updated the logic `addRow` function of `Browser` component.